src/GeneralStateTestsFiller/stEIP1153/15_tstoreCannotBeDosdFiller.yml

```diff
@@ -0,0 +1,91 @@
+# 15
+# Transient storage cannot be DOS’d: CALL(self, ...), TSTORE(x, y), TSTORE(x + 1, y), TSTORE(x + 2, y), ... ,TSTORE(x + n, y), REVERT, TLOAD(x) returns 0 and runs in under 15 seconds
+#
+# Expect slot 0 to have 0. Call fails.
+# Expect slot 1 to have 0. Loading returns 0.
+
+15_tstoreCannotBeDosd:
+  _info:
+    comment: Transient storage cannot be DOS’d.
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentNumber: 1
+    currentTimestamp: 1000
+    currentGasLimit: 0x10000000000000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 10
+
+  pre:
+    000000000000000000000000000000000000000A:
+      balance: 1000000000000000000
+      nonce: 0
+      code: |
+        :yul {
+          switch selector()
+
+          case 0x883264e8 { // doCall(uint)
+            let n:= div(calldataload(32), 0x100000000000000000000000000000000000000000000000000000000)
+            doCall(n)
+          }
+
+          case 0xf7d6c0eb { // doNStores(n)
+            let n := calldataload(32)
+            doNStores(n)
+          }
+
+          default {
+            sstore(0, 5)
+          }
+          
+          function doCall(n) {
+            mstore(0, hex"f7d6c0eb") // doNStores(n)
+            mstore(32, n)
+            let t := call(gas(), 0x000000000000000000000000000000000000000A, 0, 0, 64, 0, 0)
```

Comment:
could be more clear that it's self-calling if it's `address()` instead of `0x000000000000000000000000000000000000000A`